packages/main/src/Avatar.js

```diff
@@ -202,6 +226,14 @@ class Avatar extends UI5Element {
 
 		return null;
 	}
+
+	get accessibleNameText() {
+		if (this.accessibleName) {
+			return this.accessibleName;
+		}
+
+		return this.i18nBundle.getText(AVATAR_TOOLTIP) || undefined;
```

Comment:
The `|| undefined` part seems redundant.

Reply:
If getText function return "" empty aria-label attribute will be set.

Should I remove it?

Reply:
I think it is ok, otherwise the attribute will be set with value empty string.